Description: Do not pass --as-needed
 --as-needed was passed to gcc and produced a FTBFS with gcc 4.6
Author: Evgeni Golov <evgeni@debian.org>

--- a/fxi/src/Makefile.am
+++ b/fxi/src/Makefile.am
@@ -7,8 +7,6 @@
 
 fxi_LDADD = $(SDL_LIBS) $(COMMON_LIBS) -lz -lpng -lSDL_mixer
 
-fxi_LDFLAGS = --as-needed
-
 SOURCE_FILES = main.c dcbr.c instance.c strings.c \
               i_main.c i_debug.c i_func.c \
 	          g_main.c g_maps.c g_blit.c g_texts.c g_pal.c g_draw.c \
